reddit_io.py

```diff
@@ -68,6 +71,23 @@ def __init__(self):
 			self._positive_keywords = self._config['DEFAULT']['positive_keywords'].lower().split(',')
 		if self._config['DEFAULT']['negative_keywords']:
 			self._negative_keywords = self._config['DEFAULT']['negative_keywords'].lower().split(',')
+		if self._config['DEFAULT']['subreddit']:
+			self._subreddit = self._config['DEFAULT']['subreddit']
+		else:
+			logging.warning("Missing value of 'subreddit' in ini! Subreddit has been set to the default of r/test!")
```

Comment:
I like the logging messages, it's great, to help people debug any issues.
Together with the other changes we can use an f-string to replace the hard-coded r/test with the actual variable:
```f"<logging message> default of r/{self._subreddit}" ```
That will future-proof the logging message.

Update the other logging messages for flair and post frequency, too.